### PR TITLE
Add new params to login: tag, teaser, maxAge

### PR DESCRIPTION
**Pull request by marjuszkiewicz**
All of those params will be passed via oauth service to bff. Old flows supports teaser and tag

### CODE REVIEW

File: src/identity.js [hunk 1]
```diff
@@ -546,7 +546,13 @@ export class Identity extends EventEmitter {
      * @param {boolean} [options.preferPopup=false] - Should we try to open a popup window?
      * @param {boolean} [options.newFlow=true] - Should we try the new GDPR-safe flow or the
      * legacy/stable SPiD flow?
-     * @param {string} [options.loginHint=''] - user email hint
+     * @param {string} [options.Hint=''] - user email hint
+     * @param {string} [options.tag=''] - Pulse tag
+     * @param {string} [options.teaser=''] - Teaser slug. Teaser with given slug will be displayed
+     * in place of default teaser
+     * @param {number|string} [options.maxAge=''] - Specifies the allowable elapsed time in seconds since
```
Comment (marjuszkiewicz) [inline src/identity.js]: Not super happy about default `''`, but url builder doesn't cut nulls

**Comment by marjuszkiewicz** [comment]
To check if those params will be passed via oauth service you can use https://payment.schibsted.no/oauth/authorize?client_id=585d15cccaa79c0e30db0818&redirect_uri=https%3A%2F%2Fvglive.no&response_type=code&new-flow=true&scope=openid&state=W29iamVjdCBPYmplY3Rd&login_hint=&tag=sample-tag&teaser=sample-teaser&max_age=0

**Comment by ksimons** [comment]
+1 to making it an options object and +1 to detecting if it's an object to
avoid a breaking change.

--K.

File: src/identity.js [hunk 2]
```diff
@@ -628,27 +629,29 @@ export class Identity extends EventEmitter {
 
     /**
      * Generates the link to the new login page that'll be used in the popup or redirect flow
-     * @param {string} state - An opaque value used by the client to maintain state between the
+     * @param {object} options
+     * @param {string} [options.state] - An opaque value used by the client to maintain state between the
```
Comment (torarvid) [inline src/identity.js]: This should be `options.state` without the square brackets `[]` (they tell jsdoc that an argument is Optional, and state is Required..)

Comment (torarvid) [reply src/identity.js]: Uhm.. no.. but it's "optional with no default value", so:
```javascript
/**
@param {string} someRequiredArg
@param {string} [optionalArg]
@param {string} [optionalArgWithDefault='someDefault']
@param {string} requiredWithDefault=123
*/
```

**Pull request merged**